Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -463,14 +473,17 @@ def _async_fetch_epidata(data_source: str,
                          issues: Union[date, tuple, list],
                          lag: int,
                          time_type: str = "day") -> Union[pd.DataFrame, None]:
-    """Fetch data from Epidata API asynchronously.
+    # pylint: disable=D401
+    """[DEPRECATED] Fetch data from Epidata API asynchronously.
```

Comment:
```suggestion
    # pylint: disable=check_imperative_mood
    """[DEPRECATED] Fetch data from Epidata API asynchronously.
```
i guess im old and pylint disable doesnt accept codes anymore?